src/Libs/Ratio.elm

```diff
@@ -0,0 +1,323 @@
+module Libs.Ratio exposing
```

Comment:
Is there a way we could load this as a dependency?

Reply:
Unfortunately the [ratio lib](https://package.elm-lang.org/packages/elm-community/ratio/latest/Ratio) has not been updated to 0.19. So we can't ATM.